modules/navier_stokes/include/base/NSFVBase.h

```diff
@@ -2024,8 +2035,10 @@ NSFVBase<BaseType>::addINSMomentumFrictionKernels()
   }
   else
   {
-    const std::string kernel_type = "INSFVMomentumFriction";
+    const std::string kernel_type = "PINSFVMomentumFriction";
```

Comment:
If both of these options use the same kernel, can we somehow make this function shorter? I think there is a lot of duplication which can be deleted.

Reply:
Done